### ISSUE TITLE
Cannot boot from live DVD

### ISSUE THREAD

**Issue by Chulpo** [intialonso/intialonso.github.io#5]
My DVD rw doesn't show up in my bios when I try to choose it. It shows up when I put other live DVDs in however.   I'd really like to try this out.  Thanks for all your work.

**Comment by intialonso** [comment]
Did you verify the checksum? Can you try booting from a usb?

2016-08-03 15:51 GMT-05:00 Chulpo <notifications@github.com>:

**Comment by Chulpo** [comment]
Yes. I verified the checksum. It doesn't show up on USB either unfortunately.  I downloaded both the direct link and the torrent. Same problem.  :(

**Comment by intialonso** [comment]
Sadly I have no idea whats wrong, you are the second person to report
issues with usb booting (but in that case he was able to boot from DVD). I
will need to debug the problem and rebuild the image.

I created another USB using Rufus on Windows and it worked fine.

2016-08-04 12:53 GMT-05:00 Chulpo <notifications@github.com>:

**Comment by Chulpo** [comment]
I don't think it's anything you did. I just made a live DVD of ExTiX lxqt and have the same problem. Both UbuntuQt and ExTiX will boot on my Compaq but my Lenovo doesn't see them for some reason.

**Comment by Pepano** [comment]
The same here. Bios must be in legacy mode. In UEFI only this dvd id not display in boot device. Only legacy mode display this dvd and boot is without problem.

**Comment by intialonso** [comment]
Thanks for the report, there must be something wring with the ISO, I will
take a look.
